android/routes/ui/src/main/java/com/trafi/routes/ui/internal/RoutesViewModel.kt

```diff
@@ -28,7 +28,7 @@ internal class RoutesViewModel(private val routesApi: RoutesApi) : ViewModel() {
     }
 }
 
-internal sealed class RoutesResultState {
+sealed class RoutesResultState {
```

Comment:
Not clear if this change is meaningful 🙂

Reply:
@justasm Sorry, it left from testing